Examples/Run/Common/src/GeometryExampleBase.cpp

```diff
@@ -38,10 +39,11 @@ int processGeometry(int argc, char* argv[],
   ActsExamples::Options::addMaterialOptions(desc);
   ActsExamples::Options::addObjWriterOptions(desc);
   ActsExamples::Options::addCsvWriterOptions(desc);
-  ActsExamples::Options::addOutputOptions(desc,
-                                          ActsExamples::OutputFormat::Root |
-                                              ActsExamples::OutputFormat::Json |
-                                              ActsExamples::OutputFormat::Csv);
+  ActsExamples::Options::addJsonWriterOptions(desc);
+  ActsExamples::Options::addOutputOptions(
+      desc,
+      ActsExamples::OutputFormat::Root | ActsExamples::OutputFormat::Json |
+          ActsExamples::OutputFormat::Csv | ActsExamples::OutputFormat::Obj);
```

Comment:
That's the bugfix, FYI.

Reply:
Yes I saw that. Just to be sure, the ActsExamples::OutputFormat::Obj is not missing from any other Example file ?

Reply:
I think we need to systematically go through the Examples, I will start doing this today.

Reply:
Ok, I will approve this one at least !